src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -6968,6 +7624,18 @@ address generate_avx_ghash_processBlocks() {
       StubRoutines::x86::_left_shift_mask = base64_left_shift_mask_addr();
       StubRoutines::x86::_right_shift_mask = base64_right_shift_mask_addr();
       StubRoutines::_base64_encodeBlock = generate_base64_encodeBlock();
+      if (VM_Version::supports_avx512_vbmi()) {
+        StubRoutines::x86::_lookup_lo = base64_vbmi_lookup_lo_addr();
```

Comment:
It would be good to add base64 to these names.

Reply:
Done.